.travis.yml

```diff
@@ -5,27 +5,25 @@ node_js:
 install:
   - npm install
 script:
-  - npm test
   - npm run lint
+  - npm test
 cache:
   directories:
     - node_modules
 deploy:
   - skip_cleanup: true
     provider: npm
```

Comment:
wouldn't we prefer to delegate publishing to gradle ?

Reply:
I feel it adds a lot of extra complexity and overhead to add Gradle in the mix here.
Now it is just a standard typescript/npm project. Adding Gradle will make it more complex overall , and I don't see anything that it solves right now.